wiki/Help_centre/ro.md

```diff
@@ -0,0 +1,81 @@
+---
+tags:
+  - help
+  - issue
+  - problem
+  - trouble
+  - missing
+  - ajutor
+  - problemă
+---
+
+# Centru ajutor
+
+Ai probleme cu ceva? Suntem aici să te ajutăm! Verifică câteva soluții pentru probleme comune în bara laterală din stânga ecranului tău. Dacă problema ta nu este listată aici sau încă mai ai probleme, trimite un email la [support@ppy.sh](mailto:support@ppy.sh). De asemenea, ia în considerare postarea problemei tale în [forum-ul de ajutor](https://osu.ppy.sh/community/forums/5), unde cineva va putea să te ajute.
+
+## Secțiuni {id=sections}
+
+Alege secțiunea care se potrivește cel mai bine cu problema ta pentru a găsi o soluție potrivită cu această problemă.
+
+| Secțiune | Zonă de interes |
+| :-- | :-- |
+| [Cont](/wiki/Help_centre/Account) | suporter osu!, autentificare și accesare recuperare, modificări nume, date profil |
+| [Restricții cont](/wiki/Help_centre/Account_restrictions) | Totul despre restricții, prezentare generală, proceduri de apel, motive comune și cooldown |
+| [Beatmapping și Editor](/wiki/Help_centre/Beatmapping) | Păstrarea beatmap-urilor tale, proprietatea unui beatmap, slot-uri beatmap |
+| [Client](/wiki/Help_centre/Client) | Bug-uri și crash-uri, gameplay, conexiune, performanță |
+| [Instalare și înregistrare](/wiki/Help_centre/Installation_and_registration) | Descărcarea jocului, crearea unui cont |
+| [Magazin](/wiki/Help_centre/Store) | Produse, osu!go, osu!keyboard, osu!tablet |
+| [Interdicții turnee](/wiki/Help_centre/Tournament_bans) | Totul despre interdicții turnee, prezentare generală, motive comune și cooldown-uri |
+| [Actualizarea la lazer](/wiki/Help_centre/Upgrading_to_lazer) | Migrarea la [următoarea versiune semnificativă](/wiki/Client/Release_stream/Lazer) osu! |
+| [Site](/wiki/Help_centre/Website) | Blocharea utilizatorilor, contactarea suportului, aparența site-ului |
+
+## Ajută-ne să te ajutăm {id=diagnostics}
+
+### Fișiere jurnal {id=log-files}
+
+**Fișierele jurnal sunt înregistrări detaliate despre ce face client-ul jocului în orice moment dat. Le putem folosi pentru a determina cauza problemei tale.**
+
+Aceste fișiere sunt extrem de utile și pot face rezolvarea chiar și a problemelor foarte complicate să fie trivială.
+
+Dacă un membru din echipa de asistență ți-a cerut aceste jurnale, uite cum poți face rost de ele:
+
+1. Deschide osu!.
+2. Apasă pe butonul `Opțiuni` în meniul principal sau apasă `Ctrl` + `O`.
+3. Scrie `release` în bara de căutare rapidă. Asta te va duce la linia de release curentă folosită de client-ul tău.
```

Comment:
```suggestion
3. Scrie `release` în bara de căutare rapidă. Asta te va duce la canalul de distribuție actual folosit de client-ul tău.
```
Can potentially update to match https://crowdin.com/editor/osu-web/529/en-ro?view=comfortable#16593